src/main/java/com/demoproject/cricketapp/beans/RunEvent.java

```diff
@@ -0,0 +1,42 @@
+package com.demoproject.cricketapp.beans;
+
+import lombok.*;
```

Comment:
don't import all the Classes, please add only the ones that is required

Reply:
Updated IntelliJ settings to not automatically change imports to *.
Removed * imports in new commit